lms/urls.py

```diff
@@ -64,6 +64,9 @@
     # Enrollment API RESTful endpoints
     url(r'^api/enrollment/v1/', include('enrollment.urls')),
 
+    # Entitlement API RESTful endpoints
+    url(r'^api/entitlements/v1/', include('entitlements.api.v1.urls', namespace='entitlements_api')),
```

Comment:
The root `urls.py` should point to `api.urls`, which will handle the version routing. This way, if you add a new version, all of your work is confined to `entitlements.api`.
`url(r'^api/entitlements/', include('entitlements.api.urls', namespace='entitlements_api')),`

Reply:
See https://github.com/edx/edx-platform/blob/master/cms/djangoapps/api/urls.py.